main.go

```diff
@@ -34,6 +42,10 @@ const (
 	logFormatCLILog = "clilog"
 )
 
+type mdoxMetrics struct {
```

Comment:
I was kind of keeping the struct there since I was facing issues using just `reg` variable(metrics not registering due to how CLI operates). Will try out something else and see if I can remove.

Reply:
I've kept it for the `Print()` function now. 🙂